pulpcore/app/serializers/repository.py

```diff
@@ -214,6 +214,44 @@ class RemoteSerializer(ModelSerializer):
         allow_null=True,
         required=False,
     )
+    write_only_fields = serializers.SerializerMethodField(
+        help_text=_("List of write only fields"),
+    )
+
+    def get_write_only_fields(serializer, obj, extra_data=None):
```

Comment:
```suggestion
    def get_write_only_fields(self, obj):
```
This is a method on the serializer so it should use `self`. Also, since this method is a `SerializerMethodField` it should follow their API which only accepts one argument (besides `self`): https://www.django-rest-framework.org/api-guide/fields/#serializermethodfield